pkg/apis/migration/v1alpha1/resource.go

```diff
@@ -36,8 +36,8 @@ type MigResource interface {
 	MarkReconciled()
 	// Get whether the resource has been reconciled.
 	HasReconciled() bool
-	// Get whether the resource lives in the sandbox namespace.
-	InSandbox() bool
+	// Get whether the resource lives in the tenant namespace.
+	InTenant() bool
```

Comment:
Since `InPrivileged` isn't any more explicit I say we keep it as is or we change both of them to:
```
InPrivilegedNamespace()
InTenantNamespace()
```

Any preference?

Reply:
keep it bah!

Reply:
Jeff and I spent quite a while discussing the naming around this stuff, I'd much prefer keeping it `InTenant` and `InPrivileged`